File: samples/JavaScript/Json Store/Jsql/Find One Json.js
session.setAsync(false);

var model = "Revenge From Mars";

var sql = "SELECT P.$.brand brand"
	+" FROM pinball P"
	+" WHERE LOWER(P.$.name)=LOWER(:model)";

var object = session.openJsqlCursor().database("playground")
			.query(sql).param("model",model).findOneJson();

var brand = object["brand"];	

darwino.Utils.setText("content","Brand={0}",brand);

File: samples/Java/Json Store/Jsql/Find One Json.java
String model = "Revenge From Mars";

String sql = "SELECT P.$.brand brand"
	+" FROM pinball P"
	+" WHERE LOWER(P.$.name)=LOWER(:model)";

JsonObject json = (JsonObject)session.openJsqlCursor().database("playground")
			.query(sql).param("model",model).findOneJson();

String brand = json.getString("brand");	

_formatText("Brand={0}",brand);